libs/api/ebpf_api.cpp

```diff
@@ -206,48 +213,6 @@ ebpf_create_map_xattr(_In_ const struct bpf_create_map_attr* create_attr, _Out_
     return result;
 }
 
-ebpf_result_t
-ebpf_create_map_name(
```

Comment:
The goal is that the libbpf project should have the main code in it, and this project should only have the minimal mapping from cross-plat APIs to Windows ioctls.   ebpf_create_map_name() has no place in that since bpf_map_create() is the libbpf API, and that now calls ebpf_map_create().